frontend/src/components/navigation/menuItem.tsx

```diff
@@ -1,44 +1,44 @@
-import {Menu as AMenuItem} from "antd";
-import styled from "styled-components";
-import {ProfileFilled} from '@ant-design/icons';
+import { Menu } from 'antd';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
+const Container = styled.div`
 
- const Icon = styled(ProfileFilled)
- `
- font-size: 18px;
- width: 1em;
-    height: 1em;
-    overflow: hidden;
-    font-size: 1.5rem;
-    vertical-align: middle;
-    font-family: 'Material Icons';
-    margin-right: 10%;
-    margin-left: 10%;
-    line-height: 1;
-    letter-spacing: normal;
- `;
-const MenuItem = styled(AMenuItem)`
+ background-color:inherit;
+ color: #bdbfc7;
+ font-family: Arial;
+ font-size: 14px;
+ width: 100%;
+ height: 100vh;
```

Comment:
z height to samo, wysokość powinna być obliczana na podstawie zawartości (są od tego wyjątki, ale na pewno to nie jest ten). Także height jest też do usunięcia.